profile/README.md

```diff
@@ -13,19 +13,19 @@ Our mission is to empower developers and continue supporting open source technol
 
 [Learn about open source at Oracle](https://developer.oracle.com/open-source/)
 
-[Join Oracle's developer community](https://developer.oracle.com/)<br><br>
+[Join Oracle's developer community](https://bit.ly/odevrel_slack)<br><br>
```

Comment:
```suggestion
[Join Oracle's developer community](https://bit.ly/odevrel_slack) on Slack
```

Adding HTML in Markdown is not recommended. Also, isn't there an Oracle branded URL shortening service we can use instead of Bit.ly? One that can track how many clicks we get?

Reply:
noted about HTML tags. Re link, it's coming from corporate devrel and they are using it everywhere